pebblo/app/pebblo-ui/src/constants/constant.js

```diff
@@ -457,7 +460,7 @@ const aggregatedFindingsForAllApps = APP_DATA?.findings
       if (item) item.snippetCount += cur.snippetCount;
       else
         acc.push({
-          label: cur.labelName,
+          label: KEYWORD_MAPPING[cur?.labelName] || cur?.labelName,
```

Comment:
Can you replace these `cur` and `acc` names? these are not intuitive.

Reply:
Updated variables to more meaningful names